cmd/ec2.go

```diff
@@ -47,7 +47,7 @@ func newEC2LsCmd() *cobra.Command {
 	flags.StringP("filter-tag", "t", "Name:",
 		"Filter instances by tag, such as \"Name:app-production\". The value of tag is assumed to be a partial match",
 	)
-	flags.StringP("fields", "F", "InstanceId InstanceType PublicIpAddress PrivateIpAddress StateName LaunchTime Tag:Name", "Output fields list separated by space")
+	flags.StringP("fields", "F", "InstanceId InstanceType PublicIpAddress PrivateIpAddress StateName LaunchTime AvailabilityZone Tag:Name", "Output fields list separated by space")
```

Comment:
Although the order can be overwritten with arguments, I prefer to place the default order next to `PrivateIpAddress`.
If you have any reason for this position, please let me know.